Packs/qualys/Integrations/Qualysv2/Qualysv2.yml

```diff
@@ -8,6 +8,7 @@ configuration:
   name: url
   required: true
   type: 0
+  additionalinfo: "When using asset-tag commands, the official documentation recommends that the SERVER URL parameter should be in the following format: `https://qualysapi.<tenant>.apps.qualys.com/<end-point>`. For more details see the integration documentation."
```

Comment:
Maybe worth mentioning the format we usually use - for all the other commands. 
Something like - for format assets commands use this format, instead of this format.

Reply:
Since there are a few different possible Qulays API versions and endpoints, which every user is using based on their needs and tenants I think it will only make it confusing.
This specific API format which allows using asset-tag commands is an anomaly and thats why we should document it.